build/README.md

```diff
@@ -9,11 +9,17 @@ See all Porter related [pipelines across all repositories](https://dev.azure.com
 
 Our pipeline is broken into a few discrete builds so that we can control how and when they are triggered:
 
-* **azure-pipelines.release.yml**: Validates canary and tag releases. This can be tested in a pull request 
+* **porter-release: azure-pipelines.release.yml**: Validates canary and tag releases. This can be tested in a pull request 
   using `/azp run porter-release` though steps that require credentials will fail. [View Latest Builds](https://dev.azure.com/deislabs/porter/_build?definitionId=23)
-* **azure-pipelines.install.yml**: Validates our install scripts against canary and tag releases. [View Latest Builds](https://dev.azure.com/deislabs/porter/_build?definitionId=16)
-* **azure-pipelines.pr-automatic.yml**: Validates everything we can without a live environment. [View Latest Builds](https://dev.azure.com/deislabs/porter/_build?definitionId=6)
-* **azure-pipelines.pr-manual.yml**: Validates a pull request using a live environment. Requires manual triggering 
+* **porter-check-install: azure-pipelines.install.yml**: Validates our install scripts against canary and tag releases. [View Latest Builds](https://dev.azure.com/deislabs/porter/_build?definitionId=16)
+* **porter: azure-pipelines.pr-automatic.yml**: Validates everything we can without a live environment. [View Latest Builds](https://dev.azure.com/deislabs/porter/_build?definitionId=6)
+* **porter-integration: azure-pipelines.pr-manual.yml**: Validates a pull request using a live environment. Requires manual triggering 
```

Comment:
Yes!  Thanks for adding the names here... I seem to always forget when commenting `/azp run...`